content/en/serverless/guide/azure_app_service_linux_sidecar.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Instrument Azure App Service - Linux Containers with severless-init
+kind: guide
+further_reading:
+- link: "/integrations/azure_app_services/"
+  tag: "Documentation"
+  text: "Azure App Service"
+- link: "/integrations/azure_app_service_environment/"
+  tag: "Documentation"
+  text: "Azure App Service Environment"
+---
+
+<div class="alert alert-info">To instrument your Azure App Service containers with <code>serverless-init</code>, see <a href="/serverless/azure_app_services_container">Instrument Azure App Service - Linux Container with serverless-init</a>.</div>
```

Comment:
Isn't this the name of the page I'm already on?

Also, link doesn't seem to work